EstateReporting.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -0,0 +1,503 @@
+                                                                                                    "sa",
+                                                                                                    "thisisalongpassword123!"),
+                                                                                                    ("serviceClient", "Secret1"));
+
+            IContainerService testhostContainer = DockerHelper.SetupTestHostContainer(this.TestHostContainerName,
+                                                                                      this.Logger,
+                                                                                      "stuartferguson/testhosts",
+                                                                                      new List<INetworkService>
+                                                                                      {
+                                                                                          testNetwork
+                                                                                      },
+                                                                                      traceFolder,
+                                                                                      dockerCredentials,
+                                                                                      true);
+
+            this.Containers.AddRange(new List<IContainerService>
+                                     {
+                                         eventStoreContainer,
+                                         estateManagementContainer,
+                                         securityServiceContainer,
+                                         transactionProcessorContainer,
+                                         estateReportingContainer,
+                                         testhostContainer
+                                     });
+
+            // Cache the ports
+            this.EstateManagementApiPort = estateManagementContainer.ToHostExposedEndpoint("5000/tcp").Port;
+            this.SecurityServicePort = securityServiceContainer.ToHostExposedEndpoint("5001/tcp").Port;
+            this.EventStoreHttpPort = eventStoreContainer.ToHostExposedEndpoint("2113/tcp").Port;
+            this.TransactionProcessorPort = transactionProcessorContainer.ToHostExposedEndpoint("5002/tcp").Port;
+            this.EstateReportingPort= estateReportingContainer.ToHostExposedEndpoint("5005/tcp").Port;
+
+            // Setup the base address resolvers
+            String EstateManagementBaseAddressResolver(String api) => $"http://127.0.0.1:{this.EstateManagementApiPort}";
+            String SecurityServiceBaseAddressResolver(String api) => $"http://127.0.0.1:{this.SecurityServicePort}";
+            String TransactionProcessorBaseAddressResolver(String api) => $"http://127.0.0.1:{this.TransactionProcessorPort}";
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'api'.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6339123)